scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -259,14 +259,38 @@ package object bigtable {
           bigtableOptions.getProjectId, bigtableOptions.getInstanceId, tableId)
         self.context.testOut(output.asInstanceOf[TestIO[(ByteString, Iterable[T])]])(self)
       } else {
-        val sink = BigtableIO.write().withBigtableOptions(bigtableOptions).withTableId(tableId)
+        val sink = BigtableIO.write()
+          .withBigtableOptions(bigtableOptions)
+          .withTableId(tableId)
         self
           .map(kv => KV.of(kv._1, kv._2.asJava.asInstanceOf[java.lang.Iterable[Mutation]]))
           .applyInternal(sink)
       }
       Future.failed(new NotImplementedError("Bigtable future not implemented"))
     }
 
+    /**
+      * Enhanced version to save this SCollection as a Bigtable table. Allows for creation of
+      * bulk writes to Bigtable. Note that elements must be of type `Mutation`.
+      */
+    def saveAsBigtable(tableId: String,
+                       numOfShards: Int = 0,
+                       bigtableOptions: BigtableOptions,
+                       flushInterval: Duration = Duration.standardSeconds(1))
+                      (implicit ev: T <:< Mutation)
+    : Future[Tap[(ByteString, Iterable[Mutation])]] = {
+      if (self.context.isTest) {
+        val output = BigtableBulkOutput(
+          bigtableOptions.getProjectId, bigtableOptions.getInstanceId, tableId)
+        self.context.testOut(output.asInstanceOf[TestIO[(ByteString, Iterable[T])]])(self)
```

Comment:
You don't really need the `BigtableBulkOutput` case class right? Since the output type is still `(ByteString, Iterable[T])`, same as `BigtableOutput`. The extra `Iterable` grouping is invisible to users.